packages/firebase_crashlytics/firebase_crashlytics_platform_interface/lib/src/method_channel/method_channel_crashlytics.dart

```diff
@@ -87,9 +92,32 @@ class MethodChannelFirebaseCrashlytics extends FirebaseCrashlyticsPlatform {
     required String exception,
     required String information,
     required String reason,
+    bool fatal = false,
     List<Map<String, String>>? stackTraceElements,
   }) async {
     try {
+      if (fatal) {
```

Comment:
It might be helpful for future us to have a bit of a code comment blurb above here explaining why and what it's doing if it's fatal etc